pydoctor/test/epydoc/test_restructuredtext.py

```diff
@@ -214,6 +215,40 @@ def test_rst_directive_seealso() -> None:
         </div>"""
     assert prettify(html).strip() == prettify(expected_html).strip(), html
 
+@pytest.mark.parametrize(
+    'markup', ('epytext', 'plaintext', 'restructuredtext', 'numpy', 'google')
+    )
+def test_summary(markup:str) -> None:
+    cases = [
+        ("Single line", "Single line"), 
+        ("Single line.", "Single line."), 
+        ("Single line with period.", "Single line with period."),
+        ("""
+        Single line with period.
+        
+        @type: Also with a tag.
+        """, "Single line with period."), 
+        ("Other lines with period.\nThis is attached", "Other lines with period. This is attached"),
+        ("Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. ", 
+         "Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line. Single line..."),
+        ("Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line Single line. Single line Single line Single line ", 
```

Comment:
What do you mean?

All tests cases here are different.

Reply:
From a quick is not easy to see that is specific to each input and what functionality is tested by each input.

This is why adding some info about the input instead of "Single line."  maybe it will make the test easier to read.

but this is a minor comment

Reply:
I've added some docs, thanks.